src/components/donation/Steps/Splits/Splits.tsx

```diff
@@ -12,8 +12,8 @@ export default function Split(props: Props) {
 
   const persistedLiqSplitPct =
     init.mode === "live"
-      ? liquidSplitFromSplitStepPct
-      : init.widgetConfig?.liquidSplitPct;
+      ? liquidSplitFromSplitStepPct ?? init.config?.liquidSplitPct
```

Comment:
if splits hasn't been set yet in the multi-step process, just use config (either from db or url params - Splits form need not know about this) as default value